File: Call for Problems/callforproblems.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int excluded = 0;

    for (int i = 1; i <= n; i++) {
        int d;
        scanf("%d", &d);

        if (d % 2 != 0) {
            excluded++;
        }
    }

    printf("%d", excluded);
    return 0;
}

File: Call for Problems/callforproblems.java
import java.util.Scanner;

public class callforproblems {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        
        int excluded = 0;

        for (int i = 0; i < n; i++) {
            int d = scanner.nextInt();

            if (d % 2 != 0) {
                excluded++;
            }
        }

        System.out.println(excluded);
    }
}
